docs/components/movement-sensor/adxl345.md

```diff
@@ -129,7 +128,7 @@ Then remove and fill in the attributes as applicable to your movement sensor, ac
       "namespace": "rdk",
       "attributes": {
         "board": "local",
```

Comment:
We can remove this line, and frankly we can remove the entire board component above (lines 110-123). This example doesn't use any interrupt pins, so we no longer depend on the board at all.

Reply:
Sounds good!